source/upgrade.txt

```diff
@@ -151,6 +152,13 @@ Version 5.0 Breaking Changes
   responding in the same way as ``Decimal128`` values. In particular,
   ``BsonDecimal128.isNumber()`` now returns ``true``, and
   ``BsonDecimal128.asNumber()`` returns the equivalent ``BsonNumber``.
+.. _java-breaking-changes-v5.0-list-collections:
+
+- This driver version adds a new ``authorizedCollection`` option to the
+  ``listCollections`` command. This introduces a breaking binary compatibility
```

Comment:
The driver cannot add any options to server commands, as that is a server's prerogative. I propose the following wording

```suggestion
- This driver version adds support for a new ``authorizedCollection`` option of the
  ``listCollections`` command. This introduces a breaking binary compatibility
```

Reply:
The new wording is

> This driver version adds support for a new ``authorizedCollection`` option of the ``listCollectionsNames()`` method.

It's not what I proposed, and it is not correct: the concept of "option" is not applicable to a method.